controllers/imagehub/controllers/data_helper.go

```diff
@@ -123,3 +123,15 @@ func (r *DataHelper) genFulldataByImageName(ctx context.Context, n imagehubv1.Im
 
 	return fd, nil
 }
+
+func removeDuplicateElement(input []string) []string {
+	result := make([]string, 0, len(input))
```

Comment:
This slice is make of static length as array ,but if duplicate happen, lens will be shorter, here comes some men leaks.

Reply:
so, leave it with len 0 is batter?